GVFS/GVFS/RepairJobs/PlaceholderDatabaseRepairJob.cs

```diff
@@ -13,7 +14,7 @@ public class PlaceholderDatabaseRepairJob : RepairJob
         public PlaceholderDatabaseRepairJob(ITracer tracer, TextWriter output, GVFSEnlistment enlistment)
             : base(tracer, output, enlistment)
         {
-            this.databasePath = Path.Combine(this.Enlistment.DotGVFSRoot, GVFSConstants.DotGVFS.Databases.PlaceholderList);
+            this.databasePath = Path.Combine(this.Enlistment.DotGVFSRoot, GVFSConstants.DotGVFS.Databases.VFSForGit);

```

Comment:
This variable is no longer read, because you create `blobsizesDatabasePath` down below. Drop one?